tapdb/migrations_test.go

```diff
@@ -130,3 +132,64 @@ func TestMigrationDowngrade(t *testing.T) {
 	err := db.ExecuteMigrations(TargetLatest, WithLatestVersion(1))
 	require.ErrorIs(t, err, ErrMigrationDowngrade)
 }
+
+func TestMigration20(t *testing.T) {
```

Comment:
we should add a doc. It might describe on a high level which aspects of the version 20 migration we're testing.